discovery-provider/src/tasks/index_user_bank.py

```diff
@@ -653,21 +655,32 @@ def parse_create_token_data(data: str) -> CreateTokenAccount:
 def process_user_bank_tx_details(
     session: Session,
     redis: Redis,
-    tx_info: ConfirmedTransaction,
+    tx_info: GetTransactionResp,
     tx_sig,
     timestamp,
     challenge_event_bus: ChallengeEventBus,
 ):
-    result: TransactionInfoResult = tx_info["result"]
-    meta = result["meta"]
-    error = meta["err"]
+    result = tx_info.value
+    if not result:
+        logger.info("index_user_bank.py | No result")
+        return
+    meta = result.transaction.meta
+    if not meta:
+        logger.info("index_user_bank.py | No result meta")
```

Comment:
should this be logger.error? If we're skipping a tx due to this, something weird happened.

Reply:
yes goood call